PURL-TYPES.rst

```diff
@@ -578,6 +578,25 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+nix
+---
+
+``nix`` for Nix derivations:
```

Comment:
```suggestion
``nix`` for Nix store paths:
```

The PURL is for each store path, and we may have multiple PURL for different output paths in the same derivation.